test/utils/image/manifest.go

```diff
@@ -34,6 +35,8 @@ type RegistryList struct {
 	GoogleContainerRegistry string `yaml:"googleContainerRegistry"`
 	PrivateRegistry         string `yaml:"privateRegistry"`
 	SampleRegistry          string `yaml:"sampleRegistry"`
+	GcrReleaseRegistry      string `yaml:"gcrReleaseRegistry"`
```

Comment:
I don't see why this is necessary. 

/assign @johnSchnake

Reply:
It seems like this one came in when I rebased it, I will remove these lines

Reply:
A short grep show that this registry is actually used:

```bash
./test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml:          image: gcr.io/gke-release/csi-driver-registrar:v1.0.1-gke.0
./test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml:          image: gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver:v0.4.0-gke.0
./test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml:          image: gcr.io/gke-release/csi-attacher:v1.0.1-gke.0
./test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml:          image: gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver:v0.4.0-gke.0
```

Reply:
Since it's not listed in https://github.com/kubernetes/kubernetes/blob/master/test/utils/image/manifest.go#L28 I would remove it here (also I can't see that the files above are used some where). I will open a follow up PR if these files are not used anymore.

Reply:
and https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/csi.go#L415 uses these files so we need this one